frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -38,11 +40,17 @@
     private ConfigManager configManager;
     private WorkLoadManager wlm;
     private ConcurrentHashMap<String, Model> models;
+    private ScheduledExecutorService scheduler;
 
     private ModelManager(ConfigManager configManager, WorkLoadManager wlm) {
         this.configManager = configManager;
         this.wlm = wlm;
         models = new ConcurrentHashMap<>();
+        scheduler = Executors.newScheduledThreadPool(10);
+    }
+
+    public ScheduledExecutorService getScheduler() {
```

Comment:
This schedule doesn't have to attach to ModelManager.

Reply:
Use a static final in some utility class is better than a member of ModelManager